app/components/Calendar.tsx

```diff
@@ -5,27 +5,81 @@ import EventModal from '../components/common/eventModal';
 import RdsCalendar from '../components/common/rdsCalendar';
 // tilde operator not working so used relative imports
 
-const initialEventsList: CalEvent[] = [
+export const initialEventsList: CalEvent[] = [
   {
     id: 1,
     title: 'timed event',
+    description:
+      'Lorem ipsum dolor sit amet, consectetur adipiscing elit ut aliquam, purus sit amet luctus venenatis, lectus magna fringilla urna, porttitor rhoncus dolor purus non enim praesent elementum facilisis leo, vel.',
+    location: 'Nashville, Tennessee. U.S.',
+    type: 'public',
     start: moment().add(1, 'hour').toDate(),
     end: moment().add(2, 'hours').toDate(),
+    attendees: [
+      {
+        attendee: {
+          name: 'Ankush Dharkar',
+          email: 'attendee1@example.com',
+        },
+      },
+      {
+        attendee: {
+          name: 'Harshith Venkatesh',
+          email: 'attendee2@example.com',
+        },
+      },
+      {
+        attendee: {
+          name: 'Yash Raj',
+          email: 'attendee10@example.com',
+        },
+      },
+    ],
```

Comment:
we just have the email of the attendee for now. We can display the emails instead of names.

- type needs to be replaced by visibility. Take the latest pull from `develop`, it already has the required data

Reply:
> we just have the email of the attendee for now. We can display the emails instead of names.
> 
> * type needs to be replaced by visibility. Take the latest pull from `develop`, it already has the required data

Disagree here @iRohitGaur as displaying a user's email without their consent publicly is a bad idea. 
There are two ways I can think of to solve this:
1. Add a name field in the backend if not present already. 
2. If the backend change is a bigger task we can truncate the part in the email after `@` and show it as a username. 
Example: `attendee23@gmail.com` -> `attendee23`

Reply:
@yesyash What I have mentioned is how Google also maintains it. The `consent` part comes in our TOS and while user is creating/joining a public event.

Reply:
After a discussion with @iRohitGaur, we came to the conclusion that the email Id will be hidden for all the events present in the `Social Events` tabs and visible in the `Events` tab where we show a user's private events. 

This is because, for a user's private event, there is a high chance the user will know other attendees of the event.

Reply:
Thank you @yesyash and @iRohitGaur. I have fixed it.